docs-2.0/7.data-security/4.ssl.md

```diff
@@ -0,0 +1,64 @@
+# SSL encryption
+
+Nebula Graph supports transmission with SSL encryption between the client, Graph service, Meta service, and Storage service. This topic describes how to enable SSL encryption.
+
+## Precaution
+
+Enabling SSL encryption will slightly affect performance.
```

Comment:
performance of what here, it will be better if we can be more specific here. such as the performance of access speed.